src/main/java/org/openstreetmap/atlas/checks/atlas/predicates/TagPredicates.java

```diff
@@ -53,6 +53,26 @@ private TagPredicates()
     public static final Predicate<AtlasObject> IS_HIGHWAY_AREA = object -> Validators.hasValuesFor(
             object, HighwayTag.class) && Validators.isOfType(object, AreaTag.class, AreaTag.YES);
 
+    /**
+     * Tests if the {@link AtlasObject} has a highway tag that do not contain TERTIARY_LINK,
+     * SECONDARY_LINK, PRIMARY_LINK, TRUNK_LINK, or MOTORWAY_LINK
+     */
+    public static final Predicate<AtlasObject> IS_HIGHWAY_NOT_LINK_TYPE = object -> Validators
```

Comment:
`isLink()` method in HighwayTag can be leveraged to check if the highway type is a link type or not.

Reply:
Sure. I will take a look.

Reply:
Is this being used anymore?